# Define the goal
int32 gait_type
float64 distance
float64 time
---
# Define the result
float64 distance
float64 time
---
# Define the feedback message
int32 stage
string description
float64 distance
float64 time